mathsocwebsite/settings.py

```diff
@@ -37,7 +37,7 @@
 
 ALLOWED_HOSTS = ['*']
 
-CURRENT_TERM = 1161
+CURRENT_TERM = os.environ.get('CURRENT_TERM', 1169)
```

Comment:
Is it possible to do this through OAT or something instead of storing it as an ENV variable? Or is this temporary? Otherwise were going to need some complicated cron job to increment it